examples/lighting-app/qpg/src/AppTask.cpp

```diff
@@ -108,18 +108,17 @@ void OnTriggerIdentifyEffect(Identify * identify)
 {
     sIdentifyEffect = identify->mCurrentEffectIdentifier;
 
-    if (identify->mCurrentEffectIdentifier == Clusters::Identify::EffectIdentifierEnum::kChannelChange)
+    if (identify->mEffectVariant != Clusters::Identify::EffectVariantEnum::kDefault)
```

Comment:
This is QPG platform.... is this changes really necessary ?

Reply:
they copied the wrongful cast from my mistake, so I changed that for them too. It doesn't affect much else